### ISSUE TITLE
fetchDevices does not return devices name

### ISSUE THREAD

**Issue by gscarrone** [aws-amplify/amplify-swift#3504]
### Describe the bug

After succefully login I call fetchDevices to know name and id of each device remembered, but I found "name" field is allways null.

### Steps To Reproduce

```swift
1. login on one or more devices
2. call rememberDevice function on each one
3. call fetchDevices after login on one device


let devices = try await Amplify.Auth.fetchDevices()
```

### Expected behavior

return devices name

### Amplify Framework Version

2.25.7

### Amplify Categories

Auth

### Dependency manager

Swift PM

### Swift version

5.8

### CLI version

ND

### Xcode version

15.2

### Relevant log output

```shell
<details>
<summary>Log Messages</summary>


INSERT LOG MESSAGES HERE
```
</details>
```


### Is this a regression?

No

### Regression additional context

_No response_

### Platforms

iOS

### OS Version

iOS 16, iOS 17

### Device

iPhone 15 Pro

### Specific to simulators

_No response_

### Additional context

_No response_

**Comment by gscarrone** [comment]
It works. Let me know please when release the new version and the version

**Comment by lawmicha** [comment]
This has been released in 2.26.1 https://github.com/aws-amplify/amplify-swift/releases/tag/2.26.1
